components/match2/wikis/counterstrike/match_summary.lua

```diff
@@ -219,14 +219,21 @@ function CustomMatchSummary.getByMatchId(args)
 	end
 
 	local vods = {}
+	local secondVods = {}
+	if Logic.isNotEmpty(match.links.vod2) then
+		for _, vod2 in ipairs(match.links.vod2) do
+			secondVods[vod2[2]] = vod2[1]
```

Comment:
For readability, set vod2[1] and vod2[2] to variables.
```lua
local link, gameIndex = unpack(vod2)
```
If I've understood the indexes correctly

Reply:
Yes, they follow the same structure as any link.